dags/main_summary.py

```diff
@@ -203,15 +205,49 @@
     uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/retention.sh",
     dag=dag)
 
+client_count_daily_base = [
```

Comment:
Nit: The configuration for client count view feels out of place here since it had been isolated in a separate job file before. I'm okay with this as-is since most jobs tend to have less configuration.

Reply:
What about moving it to a separate `.py` file and importing it?

Reply:
Maybe, but it's also nice having the job definition available via the airflow dashboard.